.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2.1
+
+orbs:
+  ruby-orbs: sue445/ruby-orbs@1.3.7
+
+workflows:
+  version: 2
+
+  nightly:
+    jobs:
+      - ruby-orbs/bundle-update-pr:
+          image: "circleci/ruby:2.3.3"
+          pre-bundle-update-pr:
```

Comment:
Add `branch: introduce-continuous-bundle-update`

e.g. 

```yml
- ruby-orbs/bundle-update-pr:
    branch: introduce-continuous-bundle-update
```

Reply:
@sue445 Thanks a lot!
I will try it tomorrow.

Reply:
It was solved using `sue445/ruby-orbs@1.3.8`. :clap:
Thank you!